src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -174,13 +222,47 @@ static uct_cm_ops_t uct_rdmacm_cm_ops = {
     .ep_create        = UCS_CLASS_NEW_FUNC_NAME(uct_rdmacm_cm_ep_t)
 };
 
+static uct_iface_ops_t uct_rdmacm_cm_iface_ops = {
```

Comment:
not all functions return ucs_status_t
pls go over the return types in tl.h and use correct ucs_empty_function_XX